dev/com.ibm.ws.security.saml.websso.2.0/test/com/ibm/ws/security/saml/sso20/token/Saml20TokenImplSerializationTest.java

```diff
@@ -0,0 +1,120 @@
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        activator.start(null);
+    }
+
+    /**
+     * Test to deserialize Saml20Token_1.
+     * Validate the samlString. This is used to populate
+     * all the fields in Saml20TokenImpl during deserialization.
+     *
+     * @throws SamlException
+     */
+    @Test
+    public void deserializeSaml20Token_1() throws Exception {
+        String filename = "test-resources/ser-files/Saml20TokenImpl_1.ser";
+        FileInputStream file = new FileInputStream(filename);
+        ObjectInputStream in = new ObjectInputStream(file);
+
+        Saml20TokenImpl object = (Saml20TokenImpl) in.readObject();
+
+        in.close();
+
+        //Check serializationVersion via reflection
+        Field field = Saml20TokenImpl.class.getDeclaredField("serialVersionUID");
+        field.setAccessible(true);
+        Object value = field.get(object);
+        assertEquals("The version should be -862850937499495719L", -862850937499495719L, value);
+
+        //Check samlString
```

Comment:
Add comment explaining that you only check the saml string b/c everything else in the object is derived from the string.

Reply:
It already exists in the method comment, but I can add it here too.